dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1602,7 +1603,54 @@ public void containerTwiceIntoPageAndTemplateLayout() throws Exception{
                         .setPageMode(PageMode.LIVE)
                         .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue(html.contains("content1content2content1content2"));
+        assertTrue(html.contains("content1content2content1content2"));
+    }
+
+    @Test
+    public void pageWithSidebarShouldRender() throws Exception{
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6309437)